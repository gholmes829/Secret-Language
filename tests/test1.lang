
bool f(num arg, num arg2) {
    let l1 = 3
    let l2 = 5 * l1
    # comment here
    if l2 < l1 {
        let l3 = arg
        print(l3)
    }
    else if l2 > l1 {
        let l3 = arg2
        print(l3)
    }
    else if l2 == l1 {
        let l3 = 1
        print(l3)
    }
    else {
        l2 = l3
        print(l2)
    }

    print(l1 + l2 / (arg + 1))

    return arg == 0
}

num main() {
    let k = 8
    for i in k {
        print(i)
        while (f(i + 2, 5) and true) {
            i = i - 1
            main()
        }
    }
    return 0
}