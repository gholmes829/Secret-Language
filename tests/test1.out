0
5.0
6.0
1
5.0
4.5
2
5.0
3.6
3
5.0
3.0
4
5.0
2.5714285714285716
5
5.0
2.25
6
5.0
2.0
7
5.0
1.8
